pipeline/rhos_quota/Jenkinsfile-cleanup-env.groovy

```diff
@@ -0,0 +1,44 @@
+/*
+    Pipeline script for clean up rhos-d environment.
+*/
+// Global variables section
+
+def nodeName = "centos-7"
+def sharedLib
+
+node(nodeName){
+    stage('Install prereq') {
+        checkout([
+            $class: 'GitSCM',
+            branches: [[name: '*/master']],
+            doGenerateSubmoduleConfigurations: false,
+            extensions: [[
+                $class: 'SubmoduleOption',
+                disableSubmodules: false,
+                parentCredentials: false,
+                recursiveSubmodules: true,
+                reference: '',
+                trackingSubmodules: false
+            ]],
+            submoduleCfg: [],
+            userRemoteConfigs: [[url: 'https://github.com/red-hat-storage/cephci.git']]
+        ])
+        script {
```

Comment:
FYI, we can avoid the script tag. It is more of a logical grouping of commands. However, it is useful for declarative method. Over here, we use scripted pipelines

Reply:
Sure @psathyan, Thanks for the info